src/pages/index.js

```diff
@@ -69,29 +56,82 @@ const handleImageClick = (name, link) => {
 };
 
 const handleAddCardSubmit = (data) => {
-  const cardElement = createCard(data);
-  section.addItem(cardElement);
+  cardModalButton.textContent = "Saving...";
+  api
+    .addNewCard(data)
+    .then((result) => {
+      cardModalButton.textContent = "Save";
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.

**Please, correct it everywhere**

Reply:
* You haven't corrected my remark https://github.com/rachel-naisbett/se_project_aroundtheus/pull/1#discussion_r1592987831 


You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.

**Please, correct it everywhere**

Reply:
Please, delete it from `then` if you have it in `finally`